data/projects/implement_metrics_for_coala.md

```diff
@@ -1,34 +1,46 @@
-There is more to software quality than just passing builds -
-coala should support generating metrics for your code.
-
-A wide variety of design principles influence software quality, and some of
-them can be measured objectively by so-called
-[software metrics](https://en.wikipedia.org/wiki/Software_metric). coala
-offers a good base for the implementation or wrapping of algorithms that
-calculate these metrics from a piece of code.
-
-The goal of this project proposal should be to implement handling and
-display of software metrics in coala. To prove the concept, this project
-has to include multiple bears that calculate metrics for different
-programming languages.
+There is more to software quality than just passing builds. A wide variety
+of design principles influence software quality, and some of them can be
+measured objectively by so-called
+[software metrics](https://en.wikipedia.org/wiki/Software_metric).
+Software metrics are important because they produce quantifiable measurements
+at different stages of software development which have numerous valuable
+applications in schedule and budget planning, cost estimation, quality
+assurance testing, software debugging, software performance optimization, and
+optimal personnel task assignments. They are crucial when it comes to quality
+assurance because source code metrics for quality assurance aims at writing
+maintainable code with less bugs. coala offers a good base for the
+implementation or wrapping of algorithms that calculate these metrics from a
+piece of code. The goal of this project is to implement a module in coala that
+can produce software metrics.
+
+Picking the right metrics is important and this project aims at implementing
+an API along with a set of Bears for handling and display of software metrics
+in coala.
 
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
+* A list of software metric tools to support exists.
 * The applicant has created CLI and web UI mockups for metrics.
 * The applicant has created a [cEPs](https://coala.io/cep) for his project.
 
-##### GSOC 2017 MIDTERM
+##### CODING PHASE 1
 
-* The API for bears to yield metrics has been implemented.
-* The API is properly documented and fully tested.
+* The API for bears to yield metrics is implemented.
 * At least one Bear implementing the API exists.
 * A working (not clean or merged!) prototype of the CLI visualization
-exists.
+  exists.
 
-##### GSOC 2017 FINAL
+##### CODING PHASE 2
 
 * The coala CLI can visualize metrics.
 * The other coala binaries can handle metrics.
-* User documentation is written.
+* The API and Bears are fully documented and tested.
```

Comment:
I didn't quite understand... is it unclear from the sentence that the milestone requires that API and all the Bears are fully documented and tested ?